Cargo.toml

```diff
@@ -14,11 +14,11 @@ edition = "2018"
 vendored = ["native-tls/vendored"]
 
 [dependencies]
-bytes = "0.5"
-native-tls = "0.2"
-hyper = { version = "0.13", default-features = false, features = ["tcp"] }
-tokio = { version = "0.2" }
-tokio-tls = "0.3"
+bytes = "1.0.0"
+native-tls = "0.2.6"
+hyper = { version = "0.14.1", default-features = false, features = ["tcp", "client", "http1"] }
+tokio = "1.0.0"
+tokio-native-tls = "0.3.0"
```

Comment:
What is your reason for also pinning the patch version?

Reply:
This is not pinned. It [allows cargo to update to a compatible version](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements), which would be `0.3.{0..}`, so `0.3.19` would still be considered here.

Reply:
Uhh, that's cool, thanks for the link. Didn't know it implicitly becomes a caret requirement :)

But then my question is still "Why constrain the versions more then they were constraint before?".

Reply:
Because explicit is always better than implicit.

Reply:
When it comes to versions in `Cargo.toml`s, usually it's best to just specify the minimum that this specific crate needs. That way, users of the crate can choose to upgrade their dependency when they want (maybe the newest version has a bug and they want to wait to upgrade till an even newer version is available).

Reply:
> But then my question is still "Why constrain the versions more then they were constraint before?".

Mostly because that's what `cargo-edit`'s `upgrade` command puts there by default.

> When it comes to versions in Cargo.tomls, usually it's best to just specify the minimum that this specific crate needs. That way, users of the crate can choose to upgrade their dependency when they want (maybe the newest version has a bug and they want to wait to upgrade till an even newer version is available).

Alright, will change.

Reply:
8f4d8f9